chamge/amqp-arbiter-backend.c

```diff
@@ -7,21 +7,27 @@
 #include "config.h"
 
 #include "amqp-arbiter-backend.h"
+#include "amqp/amqp-rpc.h"
 
 #include <gio/gio.h>
-#include <amqp.h>
-#include <amqp_tcp_socket.h>
 #include <json-glib/json-glib.h>
 
 #define AMQP_ARBITER_BACKEND_SCHEMA_ID "org.hwangsaeul.Chamge1.Arbiter.AMQP"
+#define DEFAULT_CONTENT_TYPE "application/json"
+
+typedef struct _AmqpReturnArguments
+{
+  gchar *reply_queue;
+  guint correlation_id;
+} AmqpReturnAuguments;
 
 struct _ChamgeAmqpArbiterBackend
 {
   ChamgeArbiterBackend parent;
   GSettings *settings;
 
-  amqp_connection_state_t amqp_conn;
-  amqp_socket_t *amqp_socket;
+  AmqpReturnAuguments amqp_return_arg;
```

Comment:
`reply_queue` and `correlation_id` can be members here. 
but not sure if they need to be shared.
 argument passing is more maintainable than holding and sharing pointers.

Reply:
Yes. You are right. I will fix it, too.